centos_ci.sh

```diff
@@ -108,7 +108,7 @@ function perform_artifacts_upload() {
 
 function get_bundle() {
   mkdir $HOME/Downloads
-  curl -L 'https://github.com/code-ready/crc-ci-jobs/releases/download/4.0.1/crc_bundle.zip' -o $HOME/Downloads/bundle.zip
+  curl -L 'https://storage.googleapis.com/crc-bundle-github-ci/crc_bundle.zip' -o $HOME/Downloads/bundle.zip
```

Comment:
Could we use "BUNDLE_VERSION" so that it's relatively easy to know which bundle is being downloaded?

Reply:
we are using the simple name `crc_bundle.zip` to keep this link unchanged. To find out which bundle was downloaded/used you need to anyway go check the CI logs, which already shows the bundle name after unzipping.

Reply:
The link would also be mostly unchanged with curl -L "..../releases/download/$BUNDLE_VERSION/..." :)

Reply:
One more comment on that, having a non-versioned URL will get in the way if we maintain 2 branches which expect 2 different bundles.